casper/src/test/scala/coop/rchain/casper/api/BlockQueryResponseAPITest.scala

```diff
@@ -146,4 +175,45 @@ class BlockQueryResponseAPITest extends FlatSpec with Matchers with BlockStoreFi
       s"Error: Failure to find block with hash ${badTestHashQuery}"
     )
   }
+
+  "findBlockWithDeploy" should "return error when no block matching query exists" in withStore {
+    implicit blockStore =>
+      val (
+        logEff: LogStub[Id],
+        casperRef: Id[MultiParentCasperRef[Id]],
+        turanOracleEffect: SafetyOracle[Id]
+      )             = emptyEffects(blockStore)
+      val user      = ByteString.EMPTY
+      val timestamp = 0L
+      val blockQueryResponse = BlockAPI.findBlockWithDeploy[Id](user, timestamp)(
+        syncId,
+        casperRef,
+        logEff,
+        turanOracleEffect,
+        blockStore
+      )
+      blockQueryResponse.status should be(
+        s"Error: Failure to find block containing deploy signed by  with timestamp ${timestamp.toString}"
```

Comment:
Please put `${PrettyPrinter.buildString(user)}` after _signed by_ . It looks like you're missing it but it's just the pub key is empty.

Reply:
Well I intentionally put as close as possible to the actual rendering in the be(...) part. I can add `${PrettyPrinter.buildString(user)}` if you think that is clearer.

Reply:
It is, thanks.